docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,154 @@
-# Code Reviews
-
-![alt codeQuality](https://imgs.xkcd.com/comics/code_quality.png "Code Quality")
+# Code reviews
+
+![XKCD Code Quality](https://imgs.xkcd.com/comics/code_quality.png)
+[XKCD 'Code Quality', copied under CC BY-NC 2.5](https://xkcd.com/1513/)
+
+The Internet provides a wealth of material on code reviews: 
+[on the effect of code review on company culture](https://blog.fullstory.com/what-we-learned-from-google-code-reviews-arent-just-for-catching-bugs/),
+[on formal security reviews](https://www.owasp.org/images/2/2e/OWASP_Code_Review_Guide-V1_1.pdf),
+[shorter guides](https://github.com/thoughtbot/guides/tree/master/code-review),
+[longer checklists](https://www.codeproject.com/Articles/524235/Codeplusreviewplusguidelines),
+[humanized reviews](http://www.processimpact.com/articles/humanizing_reviews.html),
+[reasons for doing code reviews in the first place](https://smartbear.com/learn/code-review/why-review-code/),
+[best practices](https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/),
+[more best practices](https://www.atlassian.com/agile/software-development/code-reviews),
+[statistics on code review effectiveness for catching bugs](https://blog.codinghorror.com/code-reviews-just-do-it/),
+and [examples of code reviews gone wrong](https://blog.fogcreek.com/effective-code-reviews-9-tips-from-a-converted-skeptic/).
+Oh, and of course there is are
+[books](https://books.google.com/books/about/Peer_Reviews_in_Software.html?id=d7BQAAAAMAAJ),
+too.  Long story short, this document presents Palantir's take on code reviews.
+Organizations with deep cultural reluctance to peer reviews may want to consult
+Karl E. Wiegers' excellent essay on [Humanizing Peer Reviews](http://www.processimpact.com/articles/humanizing_reviews.html) before trying to follow this guide.
 
 This guide covers the following topics:
 
-- [What to Review](#what-to-review)
-- [Preparing Code for Review](#preparing-code-for-review)
-- [Writing Useful Commit Messages](#writing-useful-commit-messages)
-- [Doing Code Reviews](#doing-code-reviews)
-  - [Purpose](#purpose)
-  - [Implementation](#implementation)
-  - [Legibility & Style](#legibility-style)
-  - [Maintainability](#maintainability)
-  - [Review Comments: Concise and Actionable](#review-comments-concise-and-actionable)
-- [Responding to Reviews](#responding-to-reviews)
-- [Code Review Examples](#code-review-examples)
-  - [Example: Inconsistent Naming](#example-inconsistent-naming)
-  - [Example: Inconsistent Method Signatures](#example-inconsistent-method-signatures)
-  - [Example: Code Style Violation](#example-code-style-violation)
-  - [Example: Library Use](#example-library-use)
-  - [Example: Potential Bug](#example-potential-bug)
-
-We do code reviews (CRs) to **ensure high code quality** and to **reduce the
```

Comment:
This intro line is a clear delineation of the line I’d want to qualify, below.

Reply:
not quite sure what you want me to do here? i changed the intro line, have a look if this is what you were after